### PR TITLE
WIP: Fanuc devel

### PR DESCRIPTION
**Pull request by jeritgeorge**

### CODE REVIEW

File: caljob_creator/CMakeLists.txt [hunk 1]
```diff
@@ -10,6 +10,7 @@ find_package(catkin REQUIRED COMPONENTS
   std_msgs
   tf
   industrial_extrinsic_cal
+  fanuc_msgs
```
Comment (gavanderhoorn) [inline caljob_creator/CMakeLists.txt]: This package appears to be missing from this PR.

**Comment by drchrislewis** [comment]
@jeritgeorge You did not include the new action server code nor its message definition in the WIP PR

**Comment by gavanderhoorn** [comment]
Please also consider not using the name `fanuc_msgs` for that package. It's too generic.

**Comment by drchrislewis** [comment]
I believe this PR is stale. We're now going to put a node in tesseract that has an action server with a type from control_msgs so that this repo does not depend on tesseract.

**Pull request closed without merging**